doc/latex.rst

```diff
@@ -497,11 +497,22 @@ Keys that don't need to be overridden unless in special cases are:
    .. versionchanged:: 1.6
       Remove unneeded ``{}`` after ``\\hrule``.
 
+``'makeindex'``
+   "makeindex" call, the last thing before ``\begin{document}``. With
+   ``'\\usepackage[columns=1]{idxlayout}\\makeindex'`` the index will use
+   only one column. You may have to install ``idxlayout`` LaTeX package.
+
+   Default: ``'\\makeindex'``
+
 ``'printindex'``
    "printindex" call, the last thing in the file. Override if you want to
-   generate the index differently or append some content after the index. For
-   example ``'\\footnotesize\\raggedright\\printindex'`` is advisable when the
-   index is full of long entries.
+   generate the index differently, append some content after the index, or
+   change the font. As LaTeX uses two-column mode for the index it is
+   often advisable to set this key to
+   ``'\\footnotesize\\raggedright\\printindex'``. Or, to obtain a one-column
+   index, use ``'\\def\\twocolumn[#1]{#1}\\printindex'`` (this trick may fail
```

Comment:
This trick works with our default ``'manual'`` and ``'howto'`` also with Japanese, but for the latter either with ``'howto'`` or with ``uplatex``. With ``platex`` and ``'manual'`` (``jsbook.cls``) it will not work because since #8779 (support for ``hlist`` directive) we load ``multicol`` package and ``jsbook.cls`` in that case does not use ``\twocolumn`` macro in ``theindex`` environment.

I figured it was not worthwile adding these details as anyhow ``uplatex`` will soon be default.

Reply:
>I figured it was not worthwile adding these details as anyhow uplatex will soon be default.

+1